Common/Source/Devices/devLX_EOS_ERA.cpp

```diff
@@ -726,7 +726,7 @@ return false;
 ///
 //static
 BOOL DevLX_EOS_ERA::DeclareTask(PDeviceDescriptor_t d,
-  Declaration_t* lkDecl, unsigned errBufSize, TCHAR errBuf[]) {
+   Declaration_t* const lkDecl, unsigned errBufSize, TCHAR errBuf[]) {
```

Comment:
wrong
you declare "const pointer to Delaration_t" be we need "pointer to const Declaration_t"

Reply:
tips : https://isocpp.org/wiki/faq/const-correctness#const-ptr-vs-ptr-const